src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <form
+      class="form form--big"
+      action="#"
+      method="get"
+      data-qa="big"
+    >
+      <img
+        class="icon icon--big"
+        src="images/Search.svg" 
+        alt="search" 
+      >
+
+      <input
+        class="input input--big"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot"
+        data-qa="keypress"
+      >
```

Comment:
you don't need icon--big, form--big, input--big. For the big form, you just need to set the form, icon, and input classes, and change styles according to that. What about small form, you can leave the same classes. The main idea here is not to duplicate code. So you would set styles to the big form, and in the small form you would just set the style that makes them different, you would not need to duplicate all the styles to the small one 

if you don't understand what needs to be changed ask in `fe-chat` and read the checklist one more time